app/views/memberships/index.html.haml

```diff
@@ -0,0 +1,7 @@
+%header.entry-header
+  %h1.entry-title= "Alla inkomna ansökningar"
+.post-title-divider
+  %span
+.entry-content
+  - @membership.each do | info |
```

Comment:
Just spotted this.This is a collection, right?  Should be in plural: `@memberships`or better yes `@membership_applications` or `@applications`